docs/molecules/content-delivery.html

```diff
@@ -0,0 +1,15 @@
+---
+layout: default
+title: Content Delivery
+---
+
+{% include section.html
+  id = "content-delivery"
+  title = "Content Delivery"
+  vocabulary = "Content Delivery, Downloads, Resources, Nurture"
+  syntax = "These are comprised of an image (optional), a headline, a paragraph, and a download button."
+  semantics = "These are used to deliver user-requested downloads (brochures, resources, etc.)"
+  html = "content-delivery/code/content-delivery.html"
+  css = "content-delivery/code/content-delivery.css"
+  example = "content-delivery/examples/content-delivery.html"
+%}
```

Comment:
Agreed that consistency on this is good. FWIW, I prefer `-` simply because I think it looks nicer. Is there a rhyme or reason for going with either option?